pkg/asset/manifests/ingress_test.go

```diff
@@ -91,6 +93,24 @@ func TestGenerateIngerssDefaultPlacement(t *testing.T) {
 			infrastructureTopology:    configv1.HighlyAvailableTopologyMode,
 			expectedIngressPlacement:  configv1.DefaultPlacementWorkers,
 		},
+		{
+			name:                        "test setting of aws lb type to NLB",
+			installConfigBuildOptions:   []icOption{icBuild.withLBType(configv1.NLB)},
+			controlPlaneTopology:        configv1.HighlyAvailableTopologyMode,
+			infrastructureTopology:      configv1.HighlyAvailableTopologyMode,
+			expectedIngressPlacement:    configv1.DefaultPlacementWorkers,
+			expectedIngressAWSLBType:    configv1.NLB,
+			expectedIngressPlatformType: configv1.AWSPlatformType,
+		},
+		{
+			name:                        "test setting of aws lb type to Classic",
+			installConfigBuildOptions:   []icOption{icBuild.withLBType(configv1.Classic)},
+			controlPlaneTopology:        configv1.HighlyAvailableTopologyMode,
+			infrastructureTopology:      configv1.HighlyAvailableTopologyMode,
+			expectedIngressPlacement:    configv1.DefaultPlacementWorkers,
+			expectedIngressAWSLBType:    configv1.Classic,
+			expectedIngressPlatformType: configv1.AWSPlatformType,
+		},
 		{
```

Comment:
I had originally requested an update to unit tests to test for all input values here: https://github.com/openshift/installer/pull/6074/commits/fbd2ea07afc59239e66ae0eff26cd776a5c8cd41#r971050233.

As a result a test was added for when the LBType is set to "Classic". Should there be a test case here for the case when the install-config contains "" as LBType?

Reply:
This should be covered in the latest push.